### PR TITLE
Docs Content: Tag

### PR DESCRIPTION
**Pull request by edburyenegren-okta**
Updates our Overview content for Tag.

We'll need a new Anatomy diagram here @joejezowski-okta, but we can handle that separately if need be.

### CODE REVIEW

File: packages/odyssey/src/scss/components/_tag.scss [hunk 1]
```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 .ods-tag {
+  display: inline-block;
```
Comment (edburyenegren-okta) [inline packages/odyssey/src/scss/components/_tag.scss:16]: @arnoldsandoval-okta looks like this is necessary to ensure the same display we have in the docs today. Probably previous oversight on my part.

**Comment by edburyenegren-okta** [comment]
As a general note, I'm running all of our content through http://www.hemingwayapp.com/ as I do rewrites. While it's not a bespoke Voice/Tone guide, it will ensure our prose is readable and clear.

File: packages/docs/components/tag.md [hunk 2]
```diff
@@ -117,89 +120,43 @@ As stated above, Tags are used to describe an entity or object, not physically r
...
-
-<span class="is-fpo is-fpo-negative">The following is figma guidance and needs to be re-written</span>
-
-If you need more than one Tag in your design, you can option drag the one you have for a duplicate (like anything else).
-
-Take a look at the anatomy section on the overview page. Keep in mind that they have a Small Spacing unit (6px) in between them.
-
-</Description>
-
-<Visual variant="positive">
```
Comment (edburyenegren-okta) [inline packages/docs/components/tag.md:167]: One possible issue with the current layout is that we can't choose to allow Visuals to cascade alongside content.

e.g. if we wanted the visuals here to live under `## Usage` but not row break on `### Describe an object, don’t be the object`

Comment (arnoldsandoval-okta) [reply packages/docs/components/tag.md:167]: Let's chat about this in the morning. I want Joe to have the opportunity to think through the problem. I do think two column visual makes sense, especially for usage - but will defer to him @joejezowski-okta

Comment (edburyenegren-okta) [reply packages/docs/components/tag.md:167]: @arnoldsandoval-okta you comfy merging this in before we solve for any layout changes? i'd like to keep these to content-only for the sake of velocity

Comment (arnoldsandoval-okta) [reply packages/docs/components/tag.md:167]: No objection from me.

**Review by arnoldsandoval-okta — APPROVED** [review]

**Pull request merged**